packaging/flatpak/ignite.metainfo.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.ignite.Ignite</id>
+
+  <name>Ignite CLI</name>
+  <developer_name>Ignite</developer_name>
+
+  <summary>Ignite CLI is the all-in-one platform to build, launch, and maintain any crypto
+    application on a sovereign and secured blockchain.</summary>
+
+  <categories>
+    <category>Development</category>
+  </categories>
+
+  <metadata_license>Apache</metadata_license>
+  <project_license>Apache</project_license>
+
+  <keywords>
+    <keyword>ignite</keyword>
+    <keyword>cosmos</keyword>
+    <keyword>cosmos-sdk</keyword>
+    <keyword>blockchain</keyword>
+    <keyword>development</keyword>
+    <keyword>tendermint</keyword>
+  </keywords>
+
+  <recommends>
+    <control>keyboard</control>
+  </recommends>
+
+  <description>
+    <p>Ignite CLI offers everything you need to build, test, and launch your blockchain with a
+      decentralized worldwide community.</p>
+    <p>Ignite CLI is built on top of Cosmos SDK, the world's most popular blockchain framework.</p>
+    <p>Ignite CLI accelerates chain development by scaffolding everything you need so you can focus
+      on business logic.</p>
+  </description>
+
+  <launchable type="desktop-id">ignite.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/ignite/cli/v0.27.1/assets/ignite-cli.png</image>
+      <caption>How it works</caption>
+    </screenshot>
+  </screenshots>
+
+  <url type="bugtracker">https://github.com/ignite/cli/issues</url>
+  <url type="homepage">https://github.com/ignite/cli</url>
+
+  <content_rating type="oars-1.1" />
+
+  <releases>
+    <release version="28.0.0" date="2023-12-05" type="stable">
+      <description>
+        <p>Ignite CLI v28.0.0 adds support for Cosmos SDK Eden (v0.50)</p>
+      </description>
+      <url type="details">https://github.com/ignite/cli/releases/tag/v28.0.0</url>
+    </release>
+  </releases>
```

Comment:
Can we create a workflow for each release and automatically fill this page?

Reply:
Yes, that's the plan: https://github.com/ignite/cli/issues/3821